pkg/exporter/ipfix.go

```diff
@@ -319,9 +319,16 @@ func (ipf *IPFIX) sendDataRecord(log *logrus.Entry, record *flow.Record, v6 bool
 	if err != nil {
 		return err
 	}
-	err = dataSet.AddRecord(ipf.entitiesV4, templateID)
-	if err != nil {
-		return err
+	if v6 {
```

Comment:
its good practice to avoid if() {
} else {
}

Reply:
@msherif1234 Wouldn't that perform `data.Set.AddRecord` multiple times for v6?

Reply:
+1 @praveingk 
I think the good practice referred to is writing `if { ...return } ... return` instead of `if { ...return } else { ...return}`, but it doesn't apply here as not all paths return in the conditional expressions

Reply:
yeah @praveingk I was commenting in general and yes whatu have make sense
/lgtm

Reply:
I see :)

Reply:
Hi @praveingk 
so we can do this
```golang
entities := ipf.entitiesV4
if v6 {
  entities = ipf.entitiesV6
}
if err := dataSet.AddRecord(entities, templateID); err != nil {
  return err
}
```